Model/Log.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Bolt magento2 plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2020 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+namespace Bolt\Boltpay\Model;
+
+use Magento\Framework\Model\AbstractModel;
+
+/**
+ * Class Log
+ * @package Bolt\Boltpay\Model
+ */
+class Log extends AbstractModel implements \Magento\Framework\DataObject\IdentityInterface
+{
+    const CACHE_TAG = 'bolt_log';
```

Comment:
Can you use this `const` throughout?

Reply:
@matt-thomason This is following the Core Magento. $_cacheTag is a unique identifier that we declare for caching the model. See here - https://screencast.com/t/1whtN6qZx

Reply:
@johnmccombs1 This comment wasn't for `$_cacheTag`, it was for using `bolt_log` throughout since you use the string throughout a number of files.

Reply:
@johnmccombs1 This comment wasn't for `$_cacheTag`, it was for using `bolt_log` throughout since you use the string throughout a number of files.